src/SwedbankPay.Sdk/Models/Common/AccountInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace SwedbankPay.Client.Models.Common
+﻿namespace SwedbankPay.Sdk.Models.Common
```

Comment:
I don't think `Common` provide any useful information in a namespace. Everything "parent" to the namespace scope you're in is "common" to the current scope. Just removing `Models.Common` will make everything within the `Common` namespace more common by simply being more accessible since it will be implicitly included by all namespaces starting with `SwedbankPay.Sdk`. Please remove it.